config/index.ts

```diff
@@ -98,10 +98,13 @@ export function getConfigPath(
   return config_DEPRECATED.getConfigPath(path);
 }
 
-export function configFileExists(useHiddenConfig?: boolean) {
+export function configFileExists(
+  useHiddenConfig?: boolean,
+  path?: string
+): boolean {
   return useHiddenConfig
     ? newConfigFileExists()
-    : Boolean(config_DEPRECATED.getConfigPath());
+    : Boolean(config_DEPRECATED.getConfigPath(path));
```

Comment:
I'm not sure if this will actually check if the config file exists at the given path - it looks like `config_DEPRECATED` just returns the path if one is passed in